astroquery/nasa_exoplanet_archive/nasa_exoplanet_archive.py

```diff
@@ -65,27 +71,26 @@ def get_confirmed_planets_table(self, cache=True, show_progress=True,
         Returns
         -------
         table : `~astropy.table.QTable`
-            Table of exoplanet properties.
+            Table of one exoplanet's properties.
         """
-        if self._table is None or not cache:
+
+        planet_name = '\'' + planet_name + '\''
```

Comment:
it might be a little easier to read this as `"'"`, but I'm not sure about that...

Reply:
or maybe `'"{0}"'.format(planet_name)`